src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Abs.txt

```diff
@@ -74,15 +76,18 @@ true
 >> Abs("false")
 #Error(Kind=InvalidArgument)
 
+
 // ******** FUNCTION VALUE AS PARAMETERS ********
 
+>> Date(1976, 12,23) - Date(2000, 01, 01)
+-8409
+
 >> Abs(Date(1976, 12,23) - Date(2000, 01, 01))
 8409
 
 //Excel returns false, as the value of Abs(Date(2022, 10, 1)) in Excel is 44835 in General format, while in Date format its 10/1/2022
 >> Abs(Date(2022, 10, 1)) = 1664607600000
 true
```

Comment:
Not related to your changes, but this should return the same as Excel in C#...

Reply:
I need to move all these tests to exclude them
I'll update Date() code and tests in a separated PR